virtual_rainforest/models/abiotic/abiotic_model.py

```diff
@@ -154,6 +142,41 @@ def setup(self) -> None:
             "vapour_pressure_deficit_ref"
         )
 
+        # Generate initial profiles of air temperature [C], relative humidity [-],
+        # vapour pressure deficit [kPa], soil temperature [C], atmospheric pressure
+        # [kPa], and atmospheric :math:`\ce{CO2}` [ppm]
+        initial_microclimate = microclimate.run_microclimate(
+            data=self.data,
+            layer_roles=self.layer_roles,
+            time_index=0,
+            constants=AbioticSimpleConsts(),
```

Comment:
Do you supply `AbioticSimpleConsts()` instead of `self.constants` to shut `mypy` up?

Reply:
That was the issue we talked about yesterday. The model constants  `self.constants` are `AbioticConsts` but I also need `AbioticSimpleConsts`, so I wasn't sure how to load the best